---
layout: page
title: 
permalink: /dawn/
---

# DAWN: Diversity Among Wheat geNomes

[DAWN](http://crobiad.agwine.adelaide.edu.au/dawn/jbrowse/) provides access to a variety of public wheat data sets in the context of the International Wheat Genome Sequencing
Consortiumâ€™s (IWGSC) RefSeq v1.0 genome assembly [[1](http://dx.doi.org/10.1126/science.aar7191)]. This is achieved by
pre-processing the data and making it available through a [JBrowse](https://jbrowse.org/) genome browser.

## Resources

  * [JBrowse](http://crobiad.agwine.adelaide.edu.au/dawn/jbrowse/)
  * [Coordinate converter](http://crobiad.agwine.adelaide.edu.au/dawn/coord/) - convert coordinates from parts to/from whole pseudomolecules
  * Open data at figshare: [https://doi.org/10.4225/55/5a76e03723567](https://doi.org/10.4225/55/5a76e03723567)
  
## Help

  * Help ([Gitter Chat](https://gitter.im/CroBiAd/DAWN))
  * [Bug Report/Feature Request](https://github.com/CroBiAd/DAWN/issues/new)

## Screenshots

[Jbrowse](http://crobiad.agwine.adelaide.edu.au/dawn/jbrowse/?loc=chr4A_part2%3A235534556..235550464&tracks=IWGSC_v1.0_HC_genes%2CIWGSC_v1.0_markers_snp_infinium_90k%2CIWGSC_v1.0_markers_snp_axiom_820%2CAlsen_snpcoverage%2CRAC875_snpcoverage%2CYitpi_snpcoverage%2CPastor_snpcoverage%2CWyalkatchem_snpcoverage%2CWestonia_snpcoverage%2CACBarrie_snpcoverage%2CVolcanii_snpcoverage%2CBaxter_snpcoverage%2CChara_snpcoverage%2CDrysdale_snpcoverage%2CH45_snpcoverage%2CXiaoyan_snpcoverage%2CKukri_snpcoverage%2CGladius_snpcoverage%2CExcalibur_snpcoverage%2CIWGSC_RNASeq_multibigwig_Grain%2CIWGSC_RNASeq_multibigwig_Leaf&highlight=):
![DAWN: Wx-B1](/assets/Wx-B1.png)

[Coordinate converter](http://crobiad.agwine.adelaide.edu.au/dawn/coord/):
![DAWN: Wx-B1](/assets/DAWN-coord.png)


## DAWN Source Data Sets

The following data sets are available through DAWN:

  * Reference sequence
    * IWGSC RefSeq v1.0 genome assembly [[1](http://dx.doi.org/10.1126/science.aar7191)].
  * Gene Annotations
    * IWGSC v1.0
    * IWGSC v1.1
  * Markers (as part of IWGSC v1.0 annotation)
    * DArT
    * EST
    * MAS
    * SNP platforms
    * SSR
  * Whole genome shotgun data
    * 16 wheat accessions ([BPA](http://www.bioplatforms.com/wheat-sequencing/) [[2](https://doi.org/10.1111/j.1467-7652.2012.00717.x)])
    * Chinese spring (ENA: [PRJNA392179](https://www.ebi.ac.uk/ena/data/view/PRJNA392179) [[3](https://doi.org/10.1093/gigascience/gix097)])
  * Transcriptomic data
    * [INRA GDEC](https://urgi.versailles.inra.fr/files/RNASeqWheat/)
  * Exome data
    * 62 wheat accessions (ENA: [SRP032974](https://www.ebi.ac.uk/ena/data/view/SRP032974) [[4](https://doi.org/10.1186/s13059-015-0606-4)])

## External Resources

You may also find the following wheat related resources udeful:

  * [Ensembl Plants](http://plants.ensembl.org/Triticum_aestivum/Info/Index)
  * [Wheat@URGI portal](https://wheat-urgi.versailles.inra.fr/)
  * [http://www.wheat-expression.com/](http://www.wheat-expression.com/)
  * [Genetic Resources Information System (GRIS)](http://www.wheatpedigree.net/)
